src/main/java/wolox/training/models/Book.java

```diff
@@ -141,23 +156,31 @@ public String getIsbn() {
         return isbn;
     }
 
-    public void setIsbn(@NotNull String isbn) {
+    public void setIsbn(String isbn) {
+        Preconditions.checkNotNull(isbn, "ISBN is required");
+        Preconditions.checkArgument(isbn.matches(ISBN_FORMAT), "ISBN is not a valid ISBN code.");
         this.isbn = isbn;
     }
 
     @NotNull
     public List<User> getUsers() {
+        // Prevent when the list is empty to crash the APP.
+        if (users == null) {
```

Comment:
Why not initialize the list in the constructor or directly in the attribute to avoid this null check?

Reply:
Because I used to believe than hibernate initializes the list, yet it didn't, but I guess I could do that instead.